conans/cli/commands/remote.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+from conans.cli.command import conan_command, conan_subcommand, OnceArgument
+
+
+def output_remote_list_json(info, out):
+    myjson = json.dumps(info, indent=4)
+    out.writeln(myjson)
+
+
+def output_remote_list_cli(info, out):
+    for remote_info in info:
+        output_str = "{}: {} [Verify SSL: {}, Enabled: {}]".format(remote_info["name"],
+                                                                   remote_info["url"],
+                                                                   remote_info["verify"],
+                                                                   remote_info.get("enabled", True))
+        out.writeln(output_str)
+
+
+@conan_subcommand(formatters={"cli": output_remote_list_cli, "json": output_remote_list_json})
+def remote_list(*args, conan_api, parser, subparser):
+    """
+    List current remotes
+    """
+    args = parser.parse_args(*args)
+    info = [{"name": "remote1", "url": "https://someurl1", "verify": True, "enabled": False},
+            {"name": "remote2", "url": "https://someurl2", "verify": False},
+            {"name": "remote3", "url": "https://someurl3", "verify": True},
+            {"name": "remote4", "url": "https://someurl4", "verify": False, "enabled": False}]
+    return info
+
+
+@conan_subcommand()
+def remote_add(*args, conan_api, parser, subparser):
+    """
+    Add a remote
+    """
+    subparser.add_argument("remote", help="Name of the remote to add")
+    subparser.add_argument("url", help="Url for the rempote")
+    subparser.add_argument("-v", "--verify_ssl", action=OnceArgument, default="True",
```

Comment:
Maybe we want to consider this (for all commands).

For boolean args, use the ``store_true`` action, but not allow ``--verify_ssl``, ``--verify_ssl=True``, ``--verify_ssl=False``?

Reply:
I am still not sure about the interface. If it is a boolean, it is not great that I can type ``--verify-ssl=potato``. Maybe we want to try to define all boolean variables? If the default is ``True``, then we want to introduce an opt-out arg that is ``--non-ssl`` (or similar) instead?

Reply:
@memsharded what about this?

```
    subparser.add_argument('--verify-ssl', dest='verify_ssl', action='store_true')
    subparser.add_argument('--no-verify-ssl', dest='verify_ssl', action='store_false')
    subparser.set_defaults(verify_ssl=True)    
```

Reply:
A bit verbose for a boolean flag, that has a default (True). If it didn't have a default (or default=None), and the behavior of False, True would be different to None, then I agree. But having a dedicated specific argument that does nothing because it is the default behavior should probably be avoided.

Reply:
So then just this?

```
    subparser.add_argument('--no-verify-ssl', dest='verify_ssl', action='store_false')
    subparser.set_defaults(verify_ssl=True)
```

Reply:
Aren't you double negating it? If it is no-verify, then the default is False, store_true, and that disables ssl

Reply:
I think It's ok because the dest is `verify_ssl` should we use the `no_verify_ssl` var name instead?

Reply:
Finally what I did for `conan remote add`:

https://github.com/conan-io/conan/blob/abbd16422730c1710b0b6c0cd07e5a7428792fac/conans/cli/commands/remote.py#L40

And for `conan remote update`:

https://github.com/conan-io/conan/blob/abbd16422730c1710b0b6c0cd07e5a7428792fac/conans/cli/commands/remote.py#L68-L69

And the output is like this:

```
$ conan remote list
remote1: https://someurl1 [SSL: True, Enabled: False]
remote2: https://someurl2 [SSL: False, Enabled: True]
remote3: https://someurl3 [SSL: True, Enabled: True]
remote4: https://someurl4 [SSL: False, Enabled: False]
```